docs/fsharp/language-reference/import-declarations-the-open-keyword.md

```diff
@@ -27,7 +27,7 @@ open module-or-namespace-name
 ```
 
 ## Remarks
-Referencing code by using the fully qualified namespace or module path every time can create code that is hard to write, read, and maintain. Instead, you can use the `open` keyword for frequently used modules and namespaces so that when you reference a member of that module or namespace, you can use the short form of the name instead of the fully qualified name. This keyword is similar to the `using` keyword in C#, `using``namespace` in Visual C++, and `Imports` in Visual Basic.
+Referencing code by using the fully qualified namespace or module path every time can create code that is hard to write, read, and maintain. Instead, you can use the `open` keyword for frequently used modules and namespaces so that when you reference a member of that module or namespace, you can use the short form of the name instead of the fully qualified name. This keyword is similar to the `using` keyword in C#, `using`/`namespace` in Visual C++, and `Imports` in Visual Basic.
```

Comment:
I wasn't sure what the intent was since the C# and VB examples don't have the specification and my C++ made me remember that `using` and `namespace` are their own keywords as well as `using namespace`. 😄 I'll update the example, though, to using `using namespace`.